app/services/post_bookmarkable.rb

```diff
@@ -80,4 +80,19 @@ def self.after_create(guardian, bookmark, opts)
   def self.after_destroy(guardian, bookmark, opts)
     sync_topic_user_bookmarked(guardian.user, bookmark.bookmarkable.topic, opts)
   end
+
+  def self.cleanup_deleted
+    related_topics = DB.query(<<~SQL, grace_time: 3.days.ago)
```

Comment:
Should the `grace_time` be on the base class so that every `bookmarkable` use the same value?

Reply:
I thought that in some cases different bookmarkables could have different grace periods. We could always come back to it later.